example/readme_example.py

```diff
@@ -11,21 +11,27 @@ async def all_examples(api_key, your_learning_storage):
         # i will explain that learning_storage below.
 
         # check connection: never fails, just tells if we can connect the API. no auth needed.
-        online = await govee.check_connection()
```

Comment:
as an example, i wantet to show what you get. Therefore I used useless variables.